cli/command/login/cmd.go

```diff
@@ -0,0 +1,110 @@
+package login
+
+import (
+	"errors"
+	"github.com/dnephin/cobra"
+	"os"
+
+	"github.com/storageos/go-cli/cli"
+	"github.com/storageos/go-cli/cli/command"
+	"github.com/storageos/go-cli/cli/config"
+)
+
+type loginOptions struct {
+	host     string
+	username string
+	password string
+	delete   bool
+}
+
+func NewLoginCommand(storageosCli *command.StorageOSCli) *cobra.Command {
+	opt := loginOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "login [HOST]",
+		Short: "Store login credentials for a given storageos host",
+		Args:  cli.RequiresMaxArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opt.delete {
+				return runDelete(storageosCli, opt, args)
+			}
+			return runLogin(storageosCli, opt, args)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVar(&opt.host, "host", "", "The host to store the credentials for")
+	flags.Lookup("host").Hidden = true
+	flags.StringVar(&opt.username, "username", "", "The username to use for this host")
+	flags.StringVar(&opt.password, "password", "", "The username to use for this host")
+	flags.BoolVarP(&opt.delete, "delete", "d", false, "Delete entry for this host")
```

Comment:
instead of `storageos login --delete=true host1` would `storageos logout host1` make more sense?